Formula/m/msolve.rb

```diff
@@ -0,0 +1,61 @@
+class Msolve < Formula
+  desc "Library for Polynomial System Solving through Algebraic Methods"
+  homepage "https://msolve.lip6.fr"
+  url "https://github.com/algebraic-solving/msolve/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "742e84cf4d11eeadf62002623ecb7658e5d6d8c838fcf571fac06acf44252983"
+  license "GPL-2.0-or-later"
+
+  head "https://github.com/algebraic-solving/msolve.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Why do you need this? Does `brew livecheck --autobump msolve` return the wrong results if you omit this?